thrift/from_wire.go

```diff
@@ -182,3 +182,14 @@ func valueFromWire(spec compile.TypeSpec, w wire.Value) (interface{}, error) {
 	}
 	return result, nil
 }
+
+func isStringSpec(s compile.TypeSpec) bool {
```

Comment:
Filed https://github.com/thriftrw/thriftrw-go/issues/138 to make this easier in future